<testsuite name="Mocha Tests" tests="2" failures="0" errors="1" skipped="0" timestamp="Thu, 21 May 2020 15:32:49 GMT" time="0.008">
<testcase classname="Array #indexOf()" name="should return -1 when the value is not present" time="0.003"><failure>1 == -1

      + expected - actual

      -1
      +-1
      
AssertionError [ERR_ASSERTION]: 1 == -1
    at Context.&#x3C;anonymous&#x3E; (test\test.js:5:20)
    at processImmediate (internal/timers.js:445:21)</failure></testcase>
<testcase classname="Array #indexOf()" name="should return -1 when the value is not present" time="0"/>
</testsuite>
